packages/messaging/android/src/main/java/io/invertase/firebase/messaging/ReactNativeFirebaseMessagingStoreImpl.java

```diff
@@ -48,15 +48,15 @@ public void storeFirebaseMessage(RemoteMessage remoteMessage) {
   }
 
   @Override
-  public RemoteMessage getFirebaseMessage(String remoteMessageId) {
+  public WritableMap getFirebaseMessage(String remoteMessageId) {
```

Comment:
deprecated original symbol implementation could then call new implementation (that does the map) and just pass it through `remoteMessageFromReadableMap` as it does now.

Not the most important thing in the world but avoiding breaking changes + sticking to semver is such a huge win for library consumers that it is still important

Reply:
fixed